corehq/apps/app_manager/models.py

```diff
@@ -6247,6 +6247,23 @@ def to_json(self, app_names):
         return LatestEnabledBuildProfileSerializer(self, context={'app_names': app_names}).data
 
 
+class ApplicationReleaseLog(models.Model):
+    domain = models.CharField(max_length=255, null=False, default='')
+    timestamp = models.DateTimeField(auto_now_add=True)
```

Comment:
Should we name this `created_at` or something similar so its clear we are talking about the creation timestamp and not say, update?

Reply:
Yup, will change `timestamp` to `created_at` as it basically reflects the timestamp of creation of the log and also the time at which action was performed.